app/src/main/res/layout/fragment_camera.xml

```diff
@@ -18,7 +18,7 @@
         app:layout_constraintTop_toTopOf="parent" />
 
     <AutoCompleteTextView
-        android:id="@+id/userSearch"
+        android:id="@+id/userSearch1"
```

Comment:
My first thought was to delete CamerFragment and SharePostActivity. Still, I though I would finish this PR and then do another PR just to remove those files, as it would have given me some time to speak with Ilias with the new functionality. I would have asked him if my changes convinced him before removing his precedent work.

Reply:
Do you think I should remove them directly?

Reply:
Ok, I didn't know you planned on discussing them with Ilial. But then there will still be a trace of those file in github. So If there is some things that we need later, we can still find them. But I think you got the main functionality from them no ?